config.ru

```diff
@@ -5,3 +5,4 @@
 require_relative 'config/environment'
 
 run Rails.application
```

Comment:
I don't think we need both of these lines, do we?

Reply:
Looks like we do in fact need both.

Reply:
ah yeah, wasn't sure about that.  preference for which to keep?